state/indexer/sink/psql/schema.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TYPE block_event_type AS ENUM ('begin_block', 'end_block', '');
+CREATE TABLE block_events (
+    block_event_id SERIAL PRIMARY KEY,
+    key VARCHAR NOT NULL,
+    value VARCHAR NOT NULL,
+    height INTEGER NOT NULL,
+    type block_event_type
+);
+CREATE TABLE tx_results (
+tx_result_id SERIAL PRIMARY KEY,
+tx_result BYTEA NOT NULL
+);
+CREATE TABLE tx_events (
+    tx_event_id SERIAL PRIMARY KEY,
+    key VARCHAR NOT NULL,
+    value VARCHAR NOT NULL,
+    height INTEGER NOT NULL,
+    hash VARCHAR NOT NULL,
+    txid SERIAL,
+    FOREIGN KEY (txid) 
```

Comment:
```suggestion
    FOREIGN KEY (tx_result_id) 
```

Reply:
AFAIK a foreign key is a column or a group of columns in a table that reference the primary key of another table. So you can see I set a `txid` column to store the primary key when inserting the transaction bytes into the tx_result table. 
I can rename the `txid` column to `tx_result_id`